LEGO1/lego/legoomni/include/legoracecar.h

```diff
@@ -4,6 +4,39 @@
 #include "legocarraceactor.h"
 #include "legoracemap.h"
 
+struct EdgeReference {
+	char* m_name;
+	LegoPathBoundary* m_data;
+};
+
+// GLOBAL: LEGO1 0x100f0a20
+static EdgeReference g_edgeReferences[] = {
```

Comment:
Data should not be defined in a header. Please move this to the compilation unit (`.cpp` file). You can add the declaration as a private member of the class.

Reply:
Good point - I first thought that this is a global variable, but it looks like it is only accessed from `LegoRaceCar`.